cluster/gce/gci/configure-helper.sh

```diff
@@ -1606,11 +1606,14 @@ EOF
 #
 # $1 is the file to create.
 # $2: the log owner uid to set for the log file.
-# $3: the log owner gid to set for the log file.
```

Comment:
Keep this comment for the `KUBE_POD_LOG_READERS_GROUP` being empty case?

Reply:
Done.